menu.go

```diff
@@ -16,10 +16,26 @@ func NewMenu(label string, items ...*MenuItem) *Menu {
 type MenuItem struct {
 	ChildMenu *Menu
 	// Since: 2.1
-	IsQuit      bool
-	IsSeparator bool
-	Label       string
-	Action      func()
+	IsQuit       bool
+	IsSeparator  bool
+	Label        string
+	Action       func()
+	Disabled     bool
+	HasCheckmark bool
+	item         CanvasObject
+}
+
+// SetItem is used internally to be able to Refresh a menu item.
+func (i *MenuItem) SetItem(menuItem CanvasObject) {
+	i.item = menuItem
+}
+
+// Refresh refreshes the menu item.
+func (i *MenuItem) Refresh() {
+	if i.item == nil {
+		return
+	}
+	i.item.Refresh()
```

Comment:
Not a big deal, but wouldn't this be cleaner?
```suggestion
	if i.item != nil {
		i.item.Refresh()
	}
```

Reply:
It is one line less code, but is counter-indicated from the way we exit early and keep main code least-indented.

Reply:
Sure. It's no biggie.